File: MethodOL.java
import java.util.*;
class add{
    void sum(float a, float b){
        System.out.println("Sum of first two numbers is "+(a+b));
    }
    void sum(float a,float b,float c){
        System.out.println("Sum of three numbers is "+(a+b+c));
    }
}
class main{
    public static void main(String[] args){
        float a,b,c;
        add obj =new add();
        System.out.print("Enter three numbers : ");
        Scanner input =new Scanner(System.in);
        a=input.nextFloat();
        b=input.nextFloat();
        c=input.nextFloat();
        obj.sum(a,b);
        obj.sum(a,b,c);
    }
}

File: MethodOL.cpp
#include<iostream>
using namespace std;
void sum(int a,int b){
    cout<<"Sum of first two numbers is "<<(a+b)<<endl;
}
void sum(int a, int b,int c){
    cout<<"Sum of first three numbers is "<<(a+b+c)<<endl;
}
void sum(int a, int b,int c,int d){
    cout<<"Sum of four numbers is "<<(a+b+c+d)<<endl;
}
int main(){
    int a,b,c,d;
    cout<<"Enter four integers : ";
    cin>>a>>b>>c>>d;
    sum(a,b);
    sum(a,b,c);
    sum(a,b,c,d);
}
